frontend/src/components/ProductView/ProductButtons.tsx

```
@@ -0,0 +1,69 @@
+      window.open(`https://wa.me/${whatsappNumber}?text=${message}`, "_blank");
+    }
+  }, [onBuyWhatsApp, whatsappNumber, productTitle, t]);
+
+  return (
+    <div className="flex flex-col items-start gap-2 w-full">
+      
+      {/* Añadir a la bolsa */}
+      <button
+        onClick={handleAddToBag}
+        className="flex flex-row justify-center items-center w-full h-12 min-h-[48px]"
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)